tests/jquery.flot.composeImages.Test.js

```diff
@@ -0,0 +1,698 @@
+            expect(matchPixelColor(pixelData, 255, 0, 0, 255)).toBe(true);
+
+            pixelData = destinationCanvas.getContext('2d').getImageData(175, 100, 1, 1).data;
+            expect(matchPixelColor(pixelData, 0, 255, 0, 255)).toBe(true);
+
+            pixelData = destinationCanvas.getContext('2d').getImageData(50, 130, 1, 1).data;
+            expect(matchPixelColor(pixelData, 0, 0, 255, 255)).toBe(true);
+
+            //other points on circles should match the required colors, because of configured diameters
+            pixelData = destinationCanvas.getContext('2d').getImageData(220, 17, 1, 1).data;
+            expect(matchPixelColor(pixelData, 255, 0, 0, 255)).toBe(true);
+
+            pixelData = destinationCanvas.getContext('2d').getImageData(190, 114, 1, 1).data;
+            expect(matchPixelColor(pixelData, 0, 255, 0, 255)).toBe(true);
+
+            pixelData = destinationCanvas.getContext('2d').getImageData(80, 149, 1, 1).data;
+            expect(matchPixelColor(pixelData, 0, 0, 255, 255)).toBe(true);
+
+            //verify a pixel from the circle border, if it comes from a black line (almost black, because of antialiasing), as described in svg style
+            pixelData = destinationCanvas.getContext('2d').getImageData(79, 102, 1, 1).data;
+            expect(matchPixelColorWithError(pixelData, 0, 0, 0, 255, 15)).toBe(true);
+
+            done();
+        }, null);
+    });
+
+    it('should call composeImages on one canvas and an SVG, which are totally overlapped with transparency, using external CSS. The SVG has a different size than the ones from other tests. One component of the SVG is partially outside of the view area.', function (done) {
+        var sources = placeholder.html('<style type="text/css">' +
+        '#canvasSource1 {position:relative; left:-180px; top:-10px;}' +
+        '</style>' +
+        '<div id="test-container" style="width: 600px;height: 400px">' +
+        '<svg class="imgsrc" id="svgSource1" viewBox="0 0 250 150" xmlns="http://www.w3.org/2000/svg" width="250" height="150" title="svg1">' +
+            '<circle class="externalCSS" id="c1" cx="230" cy="20" r="15" style="fill:red"/>' +
+            '<circle class="externalCSS" id="c2" cx="175" cy="100" r="25" style="fill:#00FF00"/>' +
+            '<circle class="externalCSS" id="c3" cx="50" cy="130" r="40" style="fill:blue"/>' +
+        '</svg>' +
+        '<canvas class="imgsrc" id="canvasSource1" width="20" height="20" title="canvasSource1"></canvas>' +
+        '</div>' +
+        '<canvas id="myCanvas" width="150" height="150" style="border:1px solid #d3d3d3;"></canvas>'
+        ).find('.imgsrc').toArray();
+
+        var originalCanvas1 = document.getElementById("canvasSource1");
+        var destinationCanvas = document.getElementById("myCanvas");
+        var canvas1_Data; //used later
+        var canvas3_Data; //used later
+        var pixelData;//used later
```

Comment:
It's fine to group the short named variables on a single line like this:
```js
var var1, var2, var3;
```

Reply:
got rid of these variables